lab_5_scrapper/scrapper.py

```diff
@@ -176,6 +330,23 @@ def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> No
         Args:
             article_soup (bs4.BeautifulSoup): BeautifulSoup instance
         """
+        header = article_soup.find('h1')
+        if header:
+            self.article.title = header
+
+        self.article.author = 'NOT FOUND'
```

Comment:
author should be a list? what is written in docs

Reply:
yes, it should be, I fixed